php/README.md

```diff
@@ -1,14 +1,15 @@
-- [`5.4.36-apache`, `5.4-apache` (*5.4/apache/Dockerfile*)](https://github.com/docker-library/php/blob/5d49ba5861c0245da96132b011c8dbdad8c28188/5.4/apache/Dockerfile)
-- [`5.4.36-fpm`, `5.4-fpm` (*5.4/fpm/Dockerfile*)](https://github.com/docker-library/php/blob/5d49ba5861c0245da96132b011c8dbdad8c28188/5.4/fpm/Dockerfile)
-- [`5.5.20-cli`, `5.5-cli`, `5.5.20`, `5.5` (*5.5/Dockerfile*)](https://github.com/docker-library/php/blob/5d49ba5861c0245da96132b011c8dbdad8c28188/5.5/Dockerfile)
-- [`5.5.20-apache`, `5.5-apache` (*5.5/apache/Dockerfile*)](https://github.com/docker-library/php/blob/5d49ba5861c0245da96132b011c8dbdad8c28188/5.5/apache/Dockerfile)
-- [`5.5.20-fpm`, `5.5-fpm` (*5.5/fpm/Dockerfile*)](https://github.com/docker-library/php/blob/5d49ba5861c0245da96132b011c8dbdad8c28188/5.5/fpm/Dockerfile)
-- [`5.6.4-cli`, `5.6-cli`, `5-cli`, `cli`, `5.6.4`, `5.6`, `5`, `latest` (*5.6/Dockerfile*)](https://github.com/docker-library/php/blob/5d49ba5861c0245da96132b011c8dbdad8c28188/5.6/Dockerfile)
-- [`5.6.4-apache`, `5.6-apache`, `5-apache`, `apache` (*5.6/apache/Dockerfile*)](https://github.com/docker-library/php/blob/5d49ba5861c0245da96132b011c8dbdad8c28188/5.6/apache/Dockerfile)
-- [`5.6.4-fpm`, `5.6-fpm`, `5-fpm`, `fpm` (*5.6/fpm/Dockerfile*)](https://github.com/docker-library/php/blob/5d49ba5861c0245da96132b011c8dbdad8c28188/5.6/fpm/Dockerfile)
+- [`5.4.36-cli`, `5.4-cli`, `5.4.36`, `5.4` (*5.4/Dockerfile*)](https://github.com/docker-library/php/blob/c871678cc7c8dbc9277832e35f14f3534f2dd0fb/5.4/Dockerfile)
+- [`5.4.36-apache`, `5.4-apache` (*5.4/apache/Dockerfile*)](https://github.com/docker-library/php/blob/c871678cc7c8dbc9277832e35f14f3534f2dd0fb/5.4/apache/Dockerfile)
+- [`5.4.36-fpm`, `5.4-fpm` (*5.4/fpm/Dockerfile*)](https://github.com/docker-library/php/blob/c871678cc7c8dbc9277832e35f14f3534f2dd0fb/5.4/fpm/Dockerfile)
+- [`5.5.21-cli`, `5.5-cli`, `5.5.21`, `5.5` (*5.5/Dockerfile*)](https://github.com/docker-library/php/blob/c871678cc7c8dbc9277832e35f14f3534f2dd0fb/5.5/Dockerfile)
+- [`5.5.21-apache`, `5.5-apache` (*5.5/apache/Dockerfile*)](https://github.com/docker-library/php/blob/c871678cc7c8dbc9277832e35f14f3534f2dd0fb/5.5/apache/Dockerfile)
+- [`5.5.21-fpm`, `5.5-fpm` (*5.5/fpm/Dockerfile*)](https://github.com/docker-library/php/blob/c871678cc7c8dbc9277832e35f14f3534f2dd0fb/5.5/fpm/Dockerfile)
+- [`5.6.5-cli`, `5.6-cli`, `5-cli`, `cli`, `6`, `5.6`, `5`, `latest` (*5.6/Dockerfile*)](https://github.com/docker-library/php/blob/c871678cc7c8dbc9277832e35f14f3534f2dd0fb/5.6/Dockerfile)
+- [`5.6.5-apache`, `5.6-apache`, `5-apache`, `apache` (*5.6/apache/Dockerfile*)](https://github.com/docker-library/php/blob/c871678cc7c8dbc9277832e35f14f3534f2dd0fb/5.6/apache/Dockerfile)
+- [`5.6.5-fpm`, `5.6-fpm`, `5-fpm`, `fpm` (*5.6/fpm/Dockerfile*)](https://github.com/docker-library/php/blob/c871678cc7c8dbc9277832e35f14f3534f2dd0fb/5.6/fpm/Dockerfile)
```

Comment:
Was this done with `./update.sh php`?  It will be automatically replaced if it is different from what the script generates.